### DISCUSSION TITLE
Problem with distribution

### DISCUSSION THREAD

**Discussion by MDXgalaxy** [deepmodeling/deepmd-kit#1507] [Q&A]
I want to use the distribution of DeePMD. Although it can start by horovod, but each session runs a Repetitive process of the whole dataset. What can I do?

**Comment by njzjz** [comment]
I think it is the expected behavior. What do you want?

**Reply by MDXgalaxy** [reply]
Thanks for your reply, and I want to divide the whole dataset into several parts and each MPI process only runs one of them. For example, if I have 10000 data and each MPI process can run 2500 of them. But now the problem is that each MPI process will run all the 10000 data no matter how many MPI processes in total are. Could you please tell me how to implement it?
